recipe/common.php

```diff
@@ -55,12 +55,7 @@
  * Release
  */
 task('deploy:release', function () {
-    $release = 1;
-
-    $releases = env('releases_list');
-    if (!empty($releases)) {
-        $release = (int)$releases[0] + 1;
-    }
+    $release = date('YmdHis');
```

Comment:
In current implementation of this feature in PR we get this: then deploy process in sequence on every server we get +20 sec, +40 sec, +60 sec for every release on every server.